uint32  sensor_id

geometry_msgs/Vector3 rpy
geometry_msgs/Vector3 acceleration
geometry_msgs/Vector3 angular
geometry_msgs/Vector3 magnetic
